addon/components/confidentiality-plugin/redact.gts

```diff
@@ -1,13 +1,20 @@
-import Mark from '@lblod/ember-rdfa-editor/components/toolbar/mark';
+import { TemplateOnlyComponent } from '@ember/component/template-only';
 import t from 'ember-intl/helpers/t';
 import { NotVisibleIcon } from '@appuniversum/ember-appuniversum/components/icons/not-visible';
+import { SayController } from '@lblod/ember-rdfa-editor';
+import Mark from '@lblod/ember-rdfa-editor/components/toolbar/mark';
+
+interface Sig {
+  Args: {
+    controller: SayController;
+  };
+}
 
-<template>
+export const redact: TemplateOnlyComponent<Sig> = <template>
```

Comment:
Funnily enough, I'm now getting errors with this one:
```
Uncaught Error: Attempted to load a component, but there wasn't a component manager associated with the definition
```
in dummy app
did `pnpm i --frozen-lockfile` and `pnpm start`

Reply:
note: also happens when I checkout 47f6d0c, the commit before the master merge

Reply:
and also with #448 merged :/

Reply:
I wrote to @piemonkey yesterday because I was having the same error, thought maybe was some local problem

Reply:
I think it is because it needs to be a default export, not a named export for it to work.

So e.g.:
```js
const Redact: TemplateOnlyComponent<Sig> = <template>
  <Mark
    @icon={{NotVisibleIcon}}
    @title={{t 'confidentiality-plugin.redact'}}
    @mark='redacted'
    @controller={{@controller}}
  />
</template>;

export default Redact;
```

Reply:
confirmed that fixes it

Reply:
this is probably only a problem because it's used in a non-gts component

Reply:
yep